receiver/prometheusexec/testdata/main.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"time"
+)
+
+const defaultSleepTime = 5 * time.Millisecond
+
+func main() {
```

Comment:
Could you add a comment explaining that this will be used in the end to end test to check that the process is restarted?

Did you need to log anything here to allow it to verify that the process in fact is restarted multiple times?

Reply:
The test using this program doesn't test for restarts, it's done elsewhere! No need for logs here!